controllers/propagator/propagation.go

```diff
@@ -387,14 +375,23 @@ func (r *PolicyReconciler) getAllClusterDecisions(
 	}
 
 	// Process all placement bindings with subFilter:restricted
-	for _, pb := range pbsWithSubFilter {
+	for _, pb := range pbList.Items {
+		if pb.SubFilter != policiesv1.Restricted {
+			continue
+		}
```

Comment:
`pbsWithSubFilter` was removed partially because I wanted to use a pointer to the placementbinding in `r.getPolicyPlacementDecisions`, so that the PlacementBinding doesn't have to be copied by Go (it's only a 'shallow' copy, but still). Since `pb` is from the list range, I'm not sure if taking the address of it would actually change between iterations - it might just end up being a pointer to the last Item multiple times.

This avoids that possibility, and removes the allocation(s) required for that additional list.